src/media/mod.rs

```diff
@@ -273,26 +281,27 @@ impl<'a> UploadBuilder<'a> {
     /// Sets how many bytes to upload in one network call. By default this is set to 512 KiB.
     ///
     /// `UploadFuture` uses Twitter's chunked media upload under-the-hood, and this allows you to
-    /// set the size of each chunk. This is useful for noting how often a connection is
-    /// re-negotiated, but also for when the upload can be retried, as `UploadFuture` allows you to
-    /// retry on failure. Note that once `call` is invoked to turn `UploadBuiler` into
-    /// `UploadFuture`, the chunk size cannot be changed.
+    /// set the size of each chunk.
+    /// With a smaller chunk size, Twitter can "save" the data more often.
+    /// However, there's also network overhead, since each chunk needs a separate HTTP request.
+    /// Larger chunk sizes are better for stable network conditions, where you can reasonably expect a large upload to succeed.
+    /// Note that once the `UploadFuture` is created, the chunk size cannot be changed.
     pub fn chunk_size(self, chunk_size: usize) -> Self {
         UploadBuilder {
             chunk_size: Some(chunk_size),
             ..self
         }
     }
 
-    /// Applies the given alt text to an image or GIF when the image finishes uploading.
+    /// Applies the given alt text to the media when the upload is finished.
     pub fn alt_text<S: Into<Cow<'a, str>>>(self, alt_text: S) -> Self {
         UploadBuilder {
             alt_text: Some(alt_text.into()),
             ..self
         }
     }
 
-    /// Collects the built-up parameters and begins the upload process.
+    /// Starts upload process and returns `UploadFuture`.
```

Comment:
Oh sorry, i guess i wasn't all that clear on this one. I liked what you had before, i was just looking to add a little to make it flow better.

> Starts the upload process and returns a `Future` that represents it.

Reply:
ok